CONTRIBUTING.md

```diff
@@ -0,0 +1,59 @@
+# Contributing to Awesome Zarf
+
+Thanks for your interest in contributing to **Awesome Zarf**! We welcome community submissions of Zarf packages, tools, extensions, and learning resources that help others deploy software in airgapped, semi-connected, or secure environments.
+
+Please read through this guide before submitting a pull request.
+
+---
+
+## What Can I Contribute?
+
+You can contribute:
+
+- Appliance (distro) Packages - batteries included cluster configuration
+- Init Packages – Custom `zarf init` packages for bootstrapping environments
+- Application Packages – Reusable Zarf packages for deploying common apps
+- Tools & Extensions – Tools that improve or extend Zarf workflows
+- Learning Resources – Blog posts, videos, or guides that teach Zarf concepts
+
+---
+
+## Contribution Checklist
+
+Before submitting a pull request, make sure to:
+
+- Place your entry in the correct section (Init, Application, Utility, etc.)
+- Add it to the table in that section, sorted alphabetically by name
+- Include the following:
+  - Name of the item
+  - A short but helpful description
+  - Your GitHub handle or name as the maintainer or author
+  - A direct link to the repository or resource
+- Ensure the repository or link is publicly accessible
+- Prefer open source, maintained, and relevant contributions
+
+---
+
+## Not Accepted
+
+To keep the list high quality, we won’t accept:
+
+- Broken or private links
+- Duplicates of existing entries
+- Promotional content without value
+- Packages that don’t work with Zarf or lack a `zarf.yaml`
+
+---
+
+## Need Help?
+
+Open a discussion or create an issue if you're unsure about your contribution. We’re happy to help!
+
+---
+
+## License
+
+By contributing, you agree that your content will be published under a [Creative Commons Attribution 4.0 International License](./LICENSE).
```

Comment:
This feels like the correct License choice and is more appropriate in OSS than `CC BY-NC 4.0` but I would love more feedback here.